src/config/Config.ts

```diff
@@ -92,10 +90,20 @@ export default class Config implements ConfigInterface {
     this.setFrom(savedConfig);
   }
 
-  private ensureConfigDirectoryExists() {
-    if (!fs.existsSync(path.dirname(this.CONFIG_PATH))) {
-      fs.mkdirSync(path.dirname(this.CONFIG_PATH));
-    }
+  private initializeFromEnvironmentVariables() {
+    Object.keys(process.env).forEach(envKey => {
```

Comment:
On ething tho, could you filter `Object.keys(process.env)` for the ones that are in `JSON_KEYS`? There are just so many properties on that object...